src/server/engine_factory_v2.rs

```diff
@@ -30,6 +30,16 @@ fn get_id_and_suffix_from_path(path: &Path) -> (u64, u64) {
     (tablet_id, tablet_suffix)
 }
 
+impl KvEngineFactoryV2 {
+    fn create_tablet_helper(&self, id: u64, suffix: u64) -> Result<RocksEngine> {
```

Comment:
Why not inline it to `open_tablet_raw` can make all other functions call `open_tablet_raw` instead?